test/Tests/src/Semantic/Equals_Spec.enso

```diff
@@ -3,8 +3,10 @@ from Standard.Base import all
 from Standard.Test import Test, Test_Suite
 import Standard.Test.Extensions
 
+polyglot java import java.math.BigInteger as JavaBig
 polyglot java import java.nio.file.Path as JavaPath
```

Comment:
minor nitpick (and I know one of these lines was already there), but if we could have consistent naming of the Java-imported objects, IMO the `Java_Random` below shows a good approach
```suggestion
polyglot java import java.math.BigInteger as Java_BigInteger
polyglot java import java.nio.file.Path as Java_Path
```